lib/active_merchant/billing/credit_card_methods.rb

```diff
@@ -196,11 +196,10 @@ module CreditCardMethods
         589562..589562
       ]
 
-      # In addition to the BIN ranges listed here that all begin with 81, UnionPay cards
-      # include many ranges that start with 62.
-      # Prior to adding UnionPay, cards that start with 62 were all classified as Discover.
-      # Because UnionPay cards are able to run on Discover rails, this was kept the same.
+      # UnionPay cards beginning with 62 are able to run on Discover rails in the US and Canada
```

Comment:
I would remove this comment (and also adjust the tests below). The doc linked doesn't support the claim it's only the 62* range - it also mentions the 81* one :) leaving the doc itself as a source is fine.

We don't make this sort of comment for JCB either - which as discussed on Slack has a more complicated acceptance story (Discover rails in the US, Amex in Canada, Australia, New Zealand, native in Japan). The "card brand in country using other network"-concept is something that is not a defined thing in Active Merchant at the moment. That could be a useful addition but for now I would recommend to just call a UnionPay card a UnionPay card.

Nitpicking aside - code looks good!

Reply:
@bdewater that makes sense to me! I've removed this comment and consolidated the tests.